README.md

```diff
@@ -308,6 +308,13 @@ myConsumer {
 }
 ```
 
+### Payload logging or redaction
+
+By default, the client logs received delivery (on the TRACE level, unless timeout or sth happens - it's on some higher levels then) for
+better debugging experience. However, if you transfer some sensitive data and you don't want the delivery to be logged, you can easily
+turn it off by using `redactPayload = false` parameter in consumer configs (note: producer doesn't log the delivery at all, just its
```

Comment:
If you want to switch the logging of the body off, you should set the `redactPayload` to true, I guess. :thinking: 
```suggestion
turn it off by using `redactPayload = true` parameter in consumer configs (note: If set to true, producer doesn't log the delivery at all, just its
```

Reply:
You guess right :-D